src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -34,52 +34,34 @@ import kubernetesEks from 'images/kubernetes_logo_eks.png'
 
 The easiest way to install the Kubernetes integration is to use our guided install to generate a manifest. It bundles not just the integration DaemonSets, but also other New Relic Kubernetes configurations, like [Kubernetes events](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), [Prometheus OpenMetrics](/docs/integrations/prometheus-integrations/get-started/new-relic-prometheus-openmetrics-integration-kubernetes), and [New Relic log monitoring](/docs/logs).
```

Comment:
@nbaenam All references to Prometheus OpenMetrics should be replaced with the new Prometheus Agent.

https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/